src/main/java/school/hei/haapi/model/Fee.java

```diff
@@ -47,8 +49,8 @@
   @Enumerated(EnumType.STRING)
   private school.hei.haapi.endpoint.rest.model.Fee.TypeEnum type;
 
-  private int totalAmount;
-  private int remainingAmount;
+  private Integer totalAmount;
+  private Integer remainingAmount;
```

Comment:
no! use `int` as those attributes are mandatory

Reply:
In this case, the check will be done in the FeeMapper not in the FeeValidator. So ?